src/main/java/com/vg/hm/samples/usermanagementsvc/UserManagementSvcApplication.java

```diff
@@ -1,13 +1,19 @@
 package com.vg.hm.samples.usermanagementsvc;
 
+import com.netflix.hystrix.contrib.javanica.annotation.HystrixCommand;
+import com.netflix.hystrix.contrib.javanica.annotation.HystrixProperty;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.netflix.hystrix.EnableHystrix;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
 
+@EnableHystrix
```

Comment:
I think this is the only place where you need to use the @EnableHystrix annotation.